x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/view_details_button.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ViewDetailsButtonProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onClick?: (e: any) => void;
```

Comment:
does `React.MouseEvent<HTMLElement>` not work?

Reply:
It does, thank you Kevin!